tendermint/src/lite/types.rs

```diff
@@ -122,7 +122,40 @@ pub trait TrustThreshold {
     }
 }
 
-#[derive(Debug)]
+/// Requester can be used to request `SignedHeaders` and `ValidatorSet`s for a
+/// given height, e.g., by talking to a tendermint fullnode through RPC.
+pub trait Requester {
+    // TODO: consider putting this trait and the Store into a separate module / file...
```

Comment:
I think we should put some of the types into separate files.

Reply:
Would like to discuss this in a separate issue. Probably around general organization and structure of the light module.